test/ChampMarketplace.fees.test.ts

```diff
@@ -69,34 +83,39 @@ contract("Marketplace", (accounts) => {
 
   it("Should throw if marketplace fees is less than 0", async function () {
     try {
-      await marketContract.setMarketplacePercentFees(-1);
+      await marketContract.setMarketplacePercentFees(-1, 0, 0);
       assert.fail("setMarketplacePercentFees() did not throw.");
     } catch (e: any) {
       expect(e.message).to.includes("value out-of-bounds");
     }
   });
 
-  describe("Should share sale between fees receiver and seller", async function () {
+  describe("Should share sale between fees receiver, seller and burn address", async function () {
+    type FeesTuple = [number, number, number];
     type TestCase = [
       string, // Title
       number, // Sale price
-      number, // Marketplace fees
+      FeesTuple, // Marketplace fees
       number, // Expectation for seller
-      number // Expectation for fees receiver
+      number, // Expectation for fees receiver
+      number // Expectation for burn address
     ];
 
     const tests: TestCase[] = [
-      ["Common case", 100, 5, 95, 5],
-      ["Price too low for split", 1, 5, 1, 0],
-      ["Round should be in favor of seller", 99, 5, 95, 4],
-      ["0% fees", 50, 0, 50, 0],
-      ["100% fees", 50, 100, 0, 50],
+      ["Common case", 100, [4, 1, 1], 95, 5, 1],
+      ["Price too low for split", 2, [5, 5, 2], 2, 0, 0],
+      ["Round should be in favor of seller", 99, [5, 2, 2], 94, 5, 1],
+      ["0% fees", 50, [0, 0, 0], 50, 0, 0],
+      ["100% sell fees", 50, [100, 0, 0], 0, 50, 0],
+      ["100% buy fees", 50, [0, 100, 0], 50, 50, 0],
```

Comment:
Just to confirm:

100% buy fees with a 50 price:

Buyer sends 100, seller receive 50, fee receiver receive 50 ?